tools/mpremote/mpremote/main.py

```diff
@@ -341,13 +325,15 @@ def argparse_none(description):
     "--version": "version",
 }
 
-# Add "a0", "a1", ..., "u0", "u1", ..., "c0", "c1", ... as aliases
+# Add "a0", "a1", ..., "u0", "u1", ..., "c1", "c2", ... as aliases
 # for "connect /dev/ttyACMn" (and /dev/ttyUSBn, COMn) etc.
 for port_num in range(4):
     for prefix, port in [("a", "/dev/ttyACM"), ("u", "/dev/ttyUSB"), ("c", "COM")]:
-        _BUILTIN_COMMAND_EXPANSIONS["{}{}".format(prefix, port_num)] = {
-            "command": "connect {}{}".format(port, port_num),
-            "help": 'connect to serial port "{}{}"'.format(port, port_num),
+        if port_num == 0 and port == "COM":
+            continue  # skip COM0 as it does not exist
```

Comment:
mpremote.rst still mentions COM0 in the docs for the connect command, best to remove it there as well